content/docs/user-guide/interactivity/user-interface/components/working.md

```diff
@@ -50,7 +46,7 @@ You can specify elements as properties either by dragging the element into the p
 
 1. In the **Properties** panel, next to the property to specify an element for, click the pick object button.
 
-![Properties in a Lua Script component.](/images/user-guide/game_ui_editor/ui-editor-referencing-ui-elements-2.png)
+![Properties in a Lua Script component.](/images/user-guide/interactivity/user-interface/components/ui-editor-referencing-ui-elements-2.png)
```

Comment:
The ordered list numbering resets after this image. To fix this, you'll need to indent this image.